vasyl_varyshchuk/Week_1/task_1/test_get_max_value.py

```diff
@@ -0,0 +1,23 @@
+"""
+This is module for testing get_max_value() function.
+"""
+from max_of_list import get_max_value
+
+
+def test_positive():
+
+    test_data_1 = ['5', 'some text', 'nine', '9', '3.5', '12k', '9']
+    assert get_max_value(test_data_1) == 9
+
+
+def test_negative():
+
+    test_data_2 = ['five', 'two', 'seven', 'ten', 'three']
+    assert get_max_value(test_data_2) is None
+
+
+
+def test_float_values():
+
+    test_data_3 = ['25.45', '376.45', '55.3337', '54.5', '782,3']
```

Comment:
Hmm...
Does it really work with the comma instead of a dot for float number?

Reply:
No, it woks only with dot for float number. I'll correct this mistake.